credsweeper/__main__.py

```diff
@@ -198,6 +203,11 @@ def scan(args: Namespace, content_provider: FilesProvider, json_filename: Option
 
     """
     try:
+        if args.denylist_path is not None:
+            denylist = [line for line in Util.read_file(args.denylist_path) if line]
```

Comment:
```suggestion
            denylist = [line for line in Util.read_file(args.denylist_path) if line.strip()]
```
skip user lines with whitespaces only?

Reply:
> yes, only 1 empty check

It would be 1 check IF we would perform linear search

However, `x in y` for sets doing hash map based search that is O(1) in average difficulty - meaning it does not depend on the set size 
So adding `""` to set would increase set size by 1, but not the average number of operations during checking `x in y`

In worst case `x in y` would be O(n), so it would increase operations by 1. But it's really an insanely rare worst case